### PR TITLE
fix: multiple cancellation

### PR DESCRIPTION
**Pull request by mvarlic**
* validate if the order is already canceled

image

image

### CODE REVIEW

File: Controller/Transaction/CommitWebpayM22.php [hunk 1]
```diff
@@ -75,7 +75,7 @@ public function execute()
                 return $this->orderCanceledByUser($_POST['TBK_TOKEN'], $_POST['TBK_ID_SESION'], $orderStatusCanceled);
             }
             if (isset($_GET['TBK_TOKEN'])) {
-                $this->interactsWithFullLog->logWebpayPlusRetornandoDesdeTbkFujo2Error($_POST['TBK_ID_SESION']); // Logs
+                $this->interactsWithFullLog->logWebpayPlusRetornandoDesdeTbkFujo2Error($_GET['TBK_ID_SESION']); // Logs
```
Comment (mastudillot) [inline Controller/Transaction/CommitWebpayM22.php:78]: Sanitize this parameter before writing it to the log.

Comment (mvarlic) [reply Controller/Transaction/CommitWebpayM22.php:78]: to be included in another PR

**Review by tribiec — APPROVED** [review]

**Review by mastudillot — APPROVED** [review]
LGTM

**Review by isabellaahc — APPROVED** [review]
lgtm

**Pull request merged**